docs/ceph.md

```diff
@@ -0,0 +1,98 @@
+# Ceph
+
+This used to run any ceph cli command with with arbitrary args.
+
+## Examples
+
+```bash
+kubectl rook-ceph ceph status
+
+#   cluster:
+#     id:     b74c18dd-6ee3-44fe-90b5-ed12feac46a4
+#     health: HEALTH_OK
+
+#   services:
+#     mon: 3 daemons, quorum a,b,c (age 62s)
+#     mgr: a(active, since 23s)
+#     osd: 1 osds: 1 up (since 12s), 1 in (since 30s)
+
+#   data:
+#     pools:   0 pools, 0 pgs
+#     objects: 0 objects, 0 B
+#     usage:   0 B used, 0 B / 0 B avail
+#     pgs:
```

Comment:
```suggestion
#   cluster:
#     id:     b74c18dd-6ee3-44fe-90b5-ed12feac46a4
#     health: HEALTH_OK
#
#   services:
#     mon: 3 daemons, quorum a,b,c (age 62s)
#     mgr: a(active, since 23s)
#     osd: 1 osds: 1 up (since 12s), 1 in (since 30s)
#
#   data:
#     pools:   0 pools, 0 pgs
#     objects: 0 objects, 0 B
#     usage:   0 B used, 0 B / 0 B avail
#     pgs:
```

Reply:
done